integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonMiiTestUtils.java

```diff
@@ -754,6 +631,58 @@ public static Domain createDomainResourceWithAuxiliaryImage40(
     return domainCR;
   }
 
+  /**
+   * Create a domain object for a Kubernetes domain custom resource using the basic WLS image
+   * and MII auxiliary images containing the doamin or/and cluster configuration.
```

Comment:
typo doamin --> domain 
Do we support Auxiliary Image in cluster scope in Release 4.0 ?   This feature was in Release 3.x
In 4.0 we only support Aux images in  spec/configuration  scope. 

So the method name should not be createDomainResourceWithAuxiliaryImageClusterScope()